app/models/tslr_claim.rb

```diff
@@ -84,11 +84,11 @@ class TslrClaim < ApplicationRecord
   validate  :bank_account_number_must_be_eight_digits
   validate  :bank_sort_code_must_be_six_digits
 
-  before_save :update_current_school, if: :employment_status_changed?
-  before_save :normalise_trn, if: :teacher_reference_number_changed?
-  before_save :normalise_ni_number, if: :national_insurance_number_changed?
-  before_save :normalise_bank_account_number, if: :bank_account_number_changed?
-  before_save :normalise_bank_sort_code, if: :bank_sort_code_changed?
+  before_validation :update_current_school, if: :employment_status_changed?
+  before_validation :normalise_trn, if: :teacher_reference_number_changed?
```

Comment:
When we were originally working on this we briefly discussed doing these normalisations earlier in the lifecycle. However it would mean that if the user made an error, they would see the normalised version in the form which might be a bit concerning. Happy to discuss it

Reply:
Ah. I see what you mean. I guess the current school one is invisible to users, so ok to do. I'll edit to put the rest of them back.